src/coreclr/src/pal/src/libunwind_mac/include/endian.h

```diff
@@ -0,0 +1,9 @@
+// This is an incomplete & imprecice implementation of the
```

Comment:
Do we need these fake files for Mac at all? On Windows, we didn't have some of the headers / symbols / functions. But on OSX, I would expect most if not all of those being present in the platform headers.

Reply:
Some of them like elf.h doesn't exist on macOS. Others, in order to build libunwind without any changes are needed to fake it out.